app/src/main/res/values/strings.xml

```diff
@@ -592,4 +592,14 @@
     <string name="choose_instance_prompt">Choose an instance</string>
     <string name="app_language_title">App language</string>
     <string name="systems_language">System default</string>
+
+    <!--  Timer Constants  -->
+    <string name="set_timer">Set Timer</string>
+    <string name="invalid_method_call">Invalid method call for :</string>
+    <string name="already_scheduled_title">Already Scheduled</string>
+    <string name="already_scheduled_message">The timer is already scheduled for %d Minutes. Do you want to change that?</string>
+    <string name="error_setting_timer">Some error occurred while setting timer.</string>
+    <string name="player_stop_message">Player will stop at %s</string>
+    <string name="log_stopping_the_player">Timer finished: Stopping the player</string>
+    <string name="timer_select_time_message">Please select an appropriate time.</string>
```

Comment:
Please check which of these strings is unused.

Reply:
Removed two string they were removed for that Alert Dialog